dfs/registry/utils.go

```diff
@@ -57,6 +57,7 @@ func SetRegistryImage(conn client.Connection, rImage registry.Image) error {
 			return err
 		}
 		node.Image = rImage
+		glog.Infof("Adding image %s (%s) to ZK", imagepath, node.Image.UUID)
```

Comment:
This glog could be construed to mean that the image has been added, event though the request hasn't been made to zk yet.